web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -4,13 +4,15 @@ Clicking the action button next to any Virtual Machine in the List will present
 
 ![Selecting a VM](img/1-0-list.png)
 
-A modal will be presented where the user can choose which Node the VM should be migrated to.
+When `migrate` is selected the user is presented with a dialog the requires them to confirm the migration. 
 
 ![Modal migrate](img/1-2-modal-migrate.png)
 
 ![Modal view](img/1-1-modal-info.png)
 
-After the migration is kicked off, the status of the VM showing that the migration has started will be reflected in the list view.
+ After the migration is kicked off, the status of the VM showing that the migration has started will be reflected in the list view.
```

Comment:
should we be using "kicked off" or something like "started"

Reply:
Updated. 
Thanks.